docs/howto_bentoml.md

```diff
@@ -0,0 +1,295 @@
+# How to deploy a BentoML bundle to VertexAI
+
+[BentoML](https://github.com/bentoml/BentoML) is a library that allows you to build the online serving API to serve your model.
+It can be an alternative to [Tensorflow Serving](https://www.tensorflow.org/tfx/guide/serving) or [TorchServe](https://github.com/pytorch/serve/tree/master/examples).
+The purpose of this tutorial is to show you how to deploy a BentoML bundle to a VertexAI Endpoint.
+
+Along with this tutorial, an example is provided, with some useful function to perform this process in a VertexAI pipeline.
+Check it out: [docs/assets/howto_bentoml]
```

Comment:
I have an issue with this link on my side: 
<img width="661" alt="image" src="https://github.com/artefactory/vertex-pipeline-starter-kit/assets/33326907/8e111b8c-875e-4f49-a67f-5e70832e57e7">